x/participationrewards/keeper/callbacks_test.go

```diff
@@ -5,19 +5,24 @@ import (
 	"fmt"
 	"strings"
 
+	types2 "github.com/ingenuity-build/quicksilver/third-party-chains/crescent-types/liquidity/types"
```

Comment:
I'm really not keen on using types2 and types3 as aliases - we should be as verbose as possible.

`crescentliqtypes`, `umeeleveragetypes` would be preferable imo.

Reply:
Happened during the refactor for some reason, changing them back to the way they were